spec.bs

```diff
@@ -3474,6 +3474,10 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
 <div algorithm>
   The <dfn method for="InterestGroupReportingScriptRunnerGlobalScope">registerAdMacro(|name|,
   |value|)</dfn> method steps are:
+  1. [=list/For each=] [=code unit=] |c| in |name| and |value|:
+     1. [=exception/Throw=] a {{TypeError}} if both of the following conditions hold:
+        * |c| is not an unreserved percent encoding character

```

Comment:
is there something we can link for this? Or need to explain more what it means.

Reply:
This is where it originally comes from, but I don't know if it's been integrated into HTML spec somewhere: https://datatracker.ietf.org/doc/html/rfc3986#section-2.3

Reply:
Done